app/controllers/ems_common.rb

```diff
@@ -767,10 +770,22 @@ def retrieve_vmware_cloud_api_versions
     [['vCloud API 5.1', '5.1'], ['vCloud API 5.5', '5.5'], ['vCloud API 5.6', '5.6'], ['vCloud API 9.0', '9.0']]
   end
 
-  def retrieve_openstack_security_protocols
+  def retrieve_nuage_api_versions
+    [['Version 3.2', 'v3_2'], ['Version 4.0', 'v4_0']]
```

Comment:
no, not yet. @juliancheal was working on that.

For now I would go with @Ladas' suggestion